package.json

```diff
@@ -7,7 +7,9 @@
   },
   "preferGlobal": true,
   "bin": {
-    "lounge": "index.js"
+    "lounge": "index.js",
+    "thelounge": "index.js",
+    "tl": "index.js"
```

Comment:
I do not feel strongly about the alias. I thought it would be a nice-to-have to make the transition nicer, as I'm sure lots of people will not like having to type extra characters. I don't mind keeping it or removing it depending on what people think.

Reply:
I don't see a need to have this alias.

Reply:
Meh, not a need, but a convenience. Anyway, removed.